### DISCUSSION TITLE
Biometric authentication - serious vunterlability when using face

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#3018] [Features Request & Ideas | 特性请求 & 头脑风暴]
This issue is a follow-up to a [comment on reddit ](https://www.reddit.com/r/deepin/comments/tv5clu/comment/i37pw3b/?utm_source=share&utm_medium=web2x&context=3) which points out serious security risk with biometric authentication in deepin. 
Currently authentication prompt instantly turns on face scanning in order to gain root access. It's a serious problem, because user can easily by mistake give a malicious app privileged access even without knowing the prompt will pop up. A suggested solution is to add a button which will turn on face scan manually.

**Comment by BLumia** [comment]
> ...so we hope you will not use it in environments with high security requirements. --[Description of Deepin20.5 Biometric Automatic Identification System](https://www.reddit.com/r/deepin/comments/tv5clu/linux_os_description_of_deepin205_biometric/)

I'll mark this as a feature request and see if the function behavior could change :)

**Reply by ElectrcNINJA** [reply]
It would be nice to be able to have the face recognition work like it does on the login screen. It enables the login button and allows you to click it. The Facial Recognition is not the speediest of things and you loose that speedy feeling if you have to press a button then wait. It's 5 seconds of a persons life if it works the first time. As long as the dialog is not accepted automatically once a face is recognized that would be more than enough.

**Comment by meiyixiang** [comment]
This suggestion is more reflected in the interaction level.

**Comment by meiyixiang** [comment]
This suggestion is accepted, and optimization will be scheduled later
